message-index/messages/GHC-76037/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Not In Scope Error
+summary:  An identifier is not in scope
+introduced: 9.4.1
+severity: error
+---
+
+An identifier can only be used if it is in scope, i.e. if it is bound or declared previously and that binding/declaration applies to the usage location.
```

Comment:
```suggestion
An identifier can only be used if it is in scope; i.e. if it is bound or declared previously and that binding/declaration is available at the usage location.
```

I'm not sure, but I think "applies to the usage location" could be adjusted?

Reply:
Yes, I prefer this description